modules/patch_release.nf

```diff
@@ -0,0 +1,21 @@
+// Patch release
+process patch_release {
+    container "$params.patch_release_docker"
+    secret 'SYNAPSE_AUTH_TOKEN'
+
+    input:
+    val release_synid
+    val new_release_synid
+    val retracted_sample_synid
+
+    output:
+    stdout
+
+    script:
+    """
+    python3 /patch_release/patch.py \
```

Comment:
is there any other logic that might be specific for a patch release? I noticed there were notebooks per patch release here: https://github.com/Sage-Bionetworks/Genie_processing/tree/main/patch_releases

Reply:
In the past, we had other logic like changing the values, but moving forward, the more sustainable way is for sites to fix it during consortium releases.  Therefore, patches now are only reserved for removing samples or patients.

There are probably some other parameters I could add to this in a V2 of this.